app/tasks/doctor.php

```diff
@@ -135,8 +139,15 @@
 
         try {
             $mail = $register->get('smtp'); /* @var $mail \PHPMailer\PHPMailer\PHPMailer */
+            if ($receiver != '') {
```

Comment:
Let's use 
```suggestion
            if (empty($receiver )) {
```

Reply:
Done, thanks for the help. I was not aware of that keyword.